wiki/Article_styling_criteria/Formatting/fr.md

```diff
@@ -798,9 +795,58 @@ Voici un exemple de ce à quoi doit ressembler un tableau :
 | Maria | Pas de concours | Mocha |  |
 ```
 
+## Notes de bas de page
+
+Les notes de bas de page sont de courtes notes situées à la fin de la page. Elles sont utilisées pour citer des sources ou fournir des informations d'arrière-plan qui, autrement, perturberaient le contenu l'article. Les notes de bas de page peuvent contenir du texte mis en forme et des liens.
+
+Dans le osu! wiki, les notes de bas de page sont implémentées en utilisant une syntaxe spéciale (`[^identifier]`). Les notes de bas de page peuvent utiliser n'importe quel identifiant, mais elles seront automatiquement rendues en exposant avec des numéros croissants dans l'ordre de leur première apparition. Les notes de bas de page elles-mêmes doivent être placées dans une rubrique de second niveau à la fin de l'article, appelée `Références`.
```

Comment:
Thanks, I hadn't seen that on the history. Looking more in detail on Visual Studio Code, I realize that the PR that modified this line does not appear in the history of the `en.md` file. Probably because the `outdated_since` has not been changed.